### PR TITLE
Counter

### PR DESCRIPTION
**Pull request by matejoslav**

### CODE REVIEW

File: src/util/scheduler.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import Container from "typedi";
+import { CloudantService, FlashCardService } from "../services";
+
+var CronJob = require("cron").CronJob;
+
+var flashCardService: FlashCardService = Container.get("FlashCardService");
```
Comment (MadelinTom) [inline src/util/scheduler.ts:6]: How come you are using ES5 here? Should change this to ES6 syntax to remain consistent

Comment (matejoslav) [reply src/util/scheduler.ts:5]: good spot I didnt realise this :)

**Comment by MadelinTom** [comment]
Looks good!

**Review by MadelinTom — APPROVED** [review]
all good

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20210702-095448 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20210702-095448") to [dev](https://console.bluemix.net/apps/1fa69fcb-d51a-4243-bb4f-df5f44d214dc?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 51534dca3570522c020411ec9b70293171228e25

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210708-095707 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210708-095707") to [test](https://console.bluemix.net/apps/8f252b81-8c47-42f1-9a5b-85e5a6523777?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:test"), including commit 51534dca3570522c020411ec9b70293171228e25

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-prod-20210709-111250 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-prod-20210709-111250") to [prod](https://console.bluemix.net/apps/0c01691f-fa4c-4f9f-83ab-49b1a0594f66?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:prod"), including commit 51534dca3570522c020411ec9b70293171228e25

**Pull request merged**